docs/changelog.rst

```diff
@@ -113,6 +116,9 @@ v0.20.0 (UNRELEASED)
 - Switch the ``list`` command over to using
   :meth:`mopidy.core.LibraryController.get_distinct`. (Fixes: :issue:`913`)
 
+- Add support for ``toggleoutput`` command. ``mixrampdb`` and ``mixrampdelay``
+  have also been added but have not been implemented yet.
```

Comment:
This is a bit of a nitpick, but I wouldn't write "not been implemented yet" as it implies we might add support for it. At this time I have no intention of adding Gnonlin as a dependency just so people can get crossfaded songs.